internal/store/files/recog_place_store.go

```diff
@@ -80,6 +82,25 @@ type RecogPlaceStore struct {
 	// recognized with their type preceding their name in the query. This is used
 	// to handle reconNames that are also common words like "to", "me", "a".
 	CommonWordReconNameToTypes map[string][]string
+	// US Zip Codes
+	USZipCodes map[string]struct{}
```

Comment:
curious why not do this the same way as abbreviatedNameToPlaces where the value is the RecogPlaces object & instead of having a function, can just read from this map when trying to recognize a zipcode? Is it because of memory considerations?

Reply:
Yes - memory considerations. Since the function was trivial in this case, so did it this way. If there's more logic to it in the future, we can do it the other way.